Source/WebKit/NetworkProcess/soup/NetworkProcessSoup.cpp

```diff
@@ -128,6 +129,11 @@ void NetworkProcess::platformInitializeNetworkProcess(const NetworkProcessCreati
     GRefPtr<GResolver> cachedResolver = adoptGRef(webkitCachedResolverNew(adoptGRef(g_resolver_get_default())));
     g_resolver_set_default(cachedResolver.get());
 
+    if (!parameters.localhostAliasesForTesting.isEmpty()) {
+        auto overridingResolver = adoptGRef(webkitOverridingResolverNew(cachedResolver.get(), parameters.localhostAliasesForTesting));
```

Comment:
I think we can make webkitOverridingResolverNew receive a HashSet<String>&& and here pass WTFMove(parameters.localhostAliasesForTesting)